CHANGELOG.rst

```diff
@@ -5,19 +5,18 @@ ibm.ibm_zos_core Release Notes
 .. contents:: Topics
 
 
-v1.5.0-beta.1
-=============
+v1.5.0
+======
 
 Release Summary
 ---------------
 
-Release Date: '2022-11-17'
+Release Date: '2023-4-21'
```

Comment:
I was not able to find where this is displayed in the docs, would prefer to have it like '2023-04-21' but just personal preference

Reply:
Commit [here](https://github.com/ansible-collections/ibm_zos_core/pull/752/commits/1ea7be263f4970ea401881a7fc38be6ceac9387b) should resolve the conversation